packages/verifiable-credentials/src/credentials.ts

```diff
@@ -88,9 +88,25 @@ export default class Credentials {
 	 *
 	 * @param {string} vcJwt
 	 */
-	async verifyCredential(vcJwt: string): Promise<unknown> {
+	async verifyCredential(vcJwt: string): Promise<any> {
 		const resolver = this.getResolver();
-		return verifyCredential(vcJwt, resolver);
+		const decodedCredential = await verifyCredential(vcJwt, resolver);
+
+		if (decodedCredential) {
+			const payload = decodedCredential.payload
+			const vc = payload.vc
+
+			if (vc.expirationDate) {
```

Comment:
Explain why this isn't included in the schema (see https://discord.com/channels/664594912460668960/929954765021011978/942952420433080330)